EasyClangComplete.py

```diff
@@ -78,6 +78,21 @@ def run(self, edit):
             log.debug(" Nothing to clean")
 
 
+class ApplyFixitCommand(sublime_plugin.TextCommand):
+    """Command to apply fixit at cursor position."""
+
+    def run(self, edit):
```

Comment:
Ah, ok, my bad. You do need config manager that you can only get from the EasyClangComplete class. I need to think about it. I feel there must be a better way than having commands calling commands, but I need to check it.